README.md

```diff
@@ -194,6 +194,8 @@ list.files(
 
 </details>
 
+## Contributors
```

Comment:
hmm why is this showing up as a new change?

Reply:
Good catch, looks like the merging failed and it kept the old Contributors label (now it's called Contributing). I removed the old one now.